src/components/ChatLog.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import './ChatLog.css';
+import ChatEntry from './ChatEntry';
+import PropTypes from 'prop-types';
+
+const ChatLog = (props) => {
+  const chatEntries = props.entries.map((entry) => {
+    return (
+      <li className="chat-log" key={entry.id}>
+        <ChatEntry
+          id={entry.id}
+          sender={entry.sender}
+          timeStamp={entry.timeStamp}
+          body={entry.body}
+          liked={entry.liked}
+          onUpdate={props.onUpdateLiked}
+        ></ChatEntry>
+      </li>
+    );
+  });
+
+  return (
+    <section>
+      <ul>{chatEntries}</ul>
+    </section>
+  );
+};
+
+ChatLog.propTypes = {
+  entries: PropTypes.arrayOf(
+    PropTypes.shape({
```

Comment:
great use of shape to check the objects contents! Same comment about isRequired. You can either only add is required to the props that are needed to render the component or update the tests to check for all of the props